### ISSUE TITLE
[Accessibility] Switch component lacks an outline when focused

### ISSUE THREAD

**Issue by Seigiard** [cschroeter/park-ui#457]
Currently, the Switch component lacks an outline when focused, which poses challenges for accessibility and keyboard navigation.

PR with solution is here: https://github.com/cschroeter/park-ui/pull/456
